backend/src/property_handlers.py

```diff
@@ -43,12 +43,20 @@ def get_property():
                         schema: ErrorResponse
     """
     try:
```

Comment:
Use this instead:

 try:
        payload = GetPropertyRequest().load(request.get_json())
        _property = app.services.property_service().get_property(payload)

        _property_response = GetPropertyResponse()
        _property_response.pagination = PaginationResponse()
        _property_response.pagination = PaginationResponse(partial=True).load(
            {
                "results_per_page": payload.,
                "page": 1,
                "totalNumberOfResults": len(_property)
            }
        )
        _property_response.results = _property
        return jsonify(GetPropertyResponse().dump(_property_response))